requirements.txt

```diff
@@ -5,6 +5,6 @@ pytest==2.7.3
 pytest-gitignore==1.3
 pytest-mock==0.9.0
 pytest-pylint==0.4.0
-requests>=2.9.1,<3
+requests
```

Comment:
Does this promise compatibility with _all_ versions of requests? This PR doesn't test requests 1.x. And who knows what breaking changes 3.x will incur. Makes me 😟  .

But if that's the Python way, ok!